packages/astro/test/content-layer.test.js

```diff
@@ -196,7 +196,8 @@ describe('Content Layer', () => {
 		let devServer;
 		let json;
 		before(async () => {
-			devServer = await fixture.startDevServer();
+			devServer = await fixture.startDevServer({ force: true });
+			await new Promise((r) => setTimeout(r, 700));
```

Comment:
Can we use an alternative way for this? A custom timeout will likely cause flaky tests in the future.